ST_species.c

```diff
@@ -272,6 +272,32 @@ void Species_Update_Newsize(SppIndex sp, RealF newsize)
 
 	rg = Species[sp]->res_grp;
 
+	/* If relsize is less than 0, species and individual relsizes are no longer
+	   in sync. We can correct this by summing us all individual relsizes. */
+	if(Species[sp]->relsize < 0){
+		IndivType *p = Species[sp]->IndvHead;
+		// reset relsize to 0.
+	    Species[sp]->relsize = 0;
+		//loop through all individuals and sum them.
+    	while(p)
+	    {
+    	    Species[sp]->relsize += p->relsize;
+        	p = p->Next;
+    	}
```

Comment:
These calls to relsize will need to be removed when we merge in Data_Precision_Improvements, but they are necessary in this branch.